examples/exchange_admm/configs/room_3_admm.json

```diff
@@ -19,10 +19,13 @@
             "ipopt.print_level": 0
           }
         },
-                "discretization_options": {"method": "multiple_shooting"},
-"results_file": "results//admm_opt_3.csv"
+        "discretization_options": {
+          "method": "multiple_shooting",
+          "integrator": "euler"
+        },
+        "results_file": "results//admm_opt_3.csv"
       },
-            "coordinator": {
+      "coordinator": {
```

Comment:
This souldn't do anything for `"type": "agentlib_mpc.admm_local"` (only for `admm_coordinated`)